saved-sites/saved-sites-impl/src/main/res/layout/activity_bookmarks.xml

```diff
@@ -50,7 +50,7 @@
                 android:layout_height="match_parent"
                 android:layout_margin="@dimen/keyline_2"
                 android:visibility="gone"
-                app:clearActionContentDescription="@string/clearButtonContentDescription"
+                app:clearActionContentDescription="@string/searchbar_cleartextaction_content_description_default"
```

Comment:
It was `<string name="clearButtonContentDescription">Clear search input</string>`, now it’s `<string name="searchbar_cleartextaction_content_description_default" translatable="false">Collapse</string>`, is this correct?

Reply:
Yeah, it’s the same text